packages/gestalt/src/Toast/PrimaryAction.js

```diff
@@ -13,7 +13,7 @@ type Props = {|
   target?: $ElementType<ElementConfig<typeof Link>, 'target'>,
 |};
 
-export default function ToastPrimaryAction({
+export default function PrimaryAction({
```

Comment:
These changes seem unrelated?

Reply:
Unrelated but nit changes  that doesn't make much hurt to keep

Reply:
Also for consistency with primary action in SheetMobile